.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+   - "0.12.0"
+
+branches:
```

Comment:
Right, so why commit dead code :) You're making a few decisions here on behalf of the owner that might need further discussion. Rather than block the PR on travis details why not just include this later.